ci/aws-analytical-env/jobs/pr.yml

```diff
@@ -63,6 +63,8 @@ jobs:
       - get: custom-auth-lambda-release
         version: { tag: '0.0.20' }
         trigger: true
+      - get: onboarding-email-template
+        trigger: true
```

Comment:
should be `false` for PR job

Reply:
I thought that but, the one above (custom-auth-lambda-release) is set to true so I thought I must be wrong. I've changed it now.